package.json

```diff
@@ -64,6 +64,7 @@
     "jsonwebtoken-promisified": "^1.0.3",
     "jwt-simple": "^0.5.1",
     "morgan": "^1.9.1",
+    "nock": "^10.0.6",
```

Comment:
we talked to use jest :D

Reply:
I don't know why that library remains, knowing that I have removed it before.